js/signin.js

```diff
@@ -0,0 +1,114 @@
+const QUERY_SELECTOR = function (select) {
+  return document.querySelector(select);
+};
+
+const $emailInput = QUERY_SELECTOR('#emailInput');
+const $emptyEmailError = QUERY_SELECTOR('#emptyEmailError');
+const $wrongEmailError = QUERY_SELECTOR('#wrongEmailError');
+
+$emailInput.addEventListener('focusout', function (e) {
+  const emailRegEx = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/;
```

Comment:
요렇게 작성되면 좋을 것 같아요~

```suggestion
  const EMAIL_REGEX = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/;
```

추후에 정규표현식이 계속 추가될것을 생각한다면 이런식으로 작성해도 좋아요~
```js
const REGEX = {
  EMAIL: '...',
  PASSWORD: '...',
}
```

Reply:
상수를 객체로 다룰 때 JavaScript에서는 Object.freeze라는 키워드가 같이 나오는데 확인해보시면 좋을 것 같아요!